### DISCUSSION TITLE
publish the markdown files with tag 'public' only

### DISCUSSION THREAD

**Discussion by xu4wang** [vuejs/vitepress#1730] [Q&A] [answered]
Is it possible to publish only the docs with specific tag? 

For example I have many documents in docs folder, some of them are private, and some of them are public.  I will need a way to public the public documents.

I'm thinking to add a tag with value 'public' to the docs need to be published. Is there a way to tell VitePress to generate the site only with the docs with specific tag value?

BR,Austin

**Comment by brc-dd** [comment, marked as answer]
`srcExclude` in vitepress.config?

**Reply by xu4wang** [reply]
Thanks for the hint.  I end up with creating a new vitepress project `external`, and set `srcDir: '../docs/external', which points to a sub folder of the existing docs project.

```
  "scripts": {
    "docs:dev": "vitepress dev docs",
    "ext:dev": "vitepress dev external",
    "ext:build": "vitepress build external",
    "ext:preview": "vitepress preview external"
  },
```

I run `yarn docs:dev` to see all the docs, and the other 3 scripts for external publish purpose only.
